pandora_server/DEBIAN/prerm

```diff
@@ -60,10 +60,14 @@ get_distro () {
 GET_DISTRO="`get_distro`"
 DISTRO=`echo $GET_DISTRO | cut -f 1 -d ":"`
 
-rm /etc/init.d/pandora_server
-rm /etc/init.d/tentacle_serverd
-update-rc.d pandora_server remove
-update-rc.d tentacle_serverd remove
+if [ -x `command -v systemctl` ]; then
+  systemctl disable pandora_server
+  systemctl disable tentacle_serverd
+  systemctl daemon-reload
+else
```

Comment:
Hi Cosme,
Shouldn't you have kept the calls to _rm_ (lines 63 and 64 of the original file)? From the man page of _update-rc.d_: 
> When invoked with the remove option, update-rc.d removes any links in the /etc/rcrunlevel.d directories to the script /etc/init.d/name.  The script must have been deleted already.  If the script is still present then update-rc.d aborts with an error message.

The funny thing is I tried it and got no error :-? The rest looks great as far as I can tell. Thanks for the pull request!